src/components/Icon/components/Tick.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { IconSVGProps } from '../Icon';
+
+const SvgTick = (props: IconSVGProps) => (
+  <svg viewBox="0 0 24 24" {...props}>
+    <path
+      id="_3rthkb6hpa"
```

Comment:
Think this ID can be removed, it's not used (as far as I can see?)

Reply:
Ah yes - thanks